wondrous-app/components/Common/Task/modal.tsx

```diff
@@ -986,6 +1034,18 @@ export const TaskViewModal = (props: ITaskListModalProps) => {
     }
   };
 
+  // assemble Snapshot URL & open window w/ proposal
+  const openSnapshot = async () => {
+    try {
+      const space = snapshot.key
+      const proposal = fetchedTask?.snapshotProposal
+      const url = `https://${isTest && 'demo'}.snapshot.org/#/${space}/proposal/${proposal}`
+      window.open(url)
```

Comment:
Would this open in a new tab?

Reply:
yes -- it opens up the proposal on snapshot's website

fyi the `isTest` variable is returned from the `useSnapshot` hook to reflect whether the snapshot being queried is on testnet or mainnet

Reply:
Ah awesome!